src/extension.ts

```diff
@@ -73,6 +75,29 @@ export async function activate(context: ExtensionContext) {
       }
     });
   });
+
+  if (!hasShownQuteValidationPopUp()) {
```

Comment:
At first all code which manages Qute should be stored inside https://github.com/redhat-developer/vscode-quarkus/tree/master/src/qute

I think popop check should be done https://github.com/redhat-developer/vscode-quarkus/blob/46d2781997d7b5fbeeafca3e7deb491934218641/src/qute/languageServer/client.ts#L77 (after that Qute LS is started). 

Indeed this popup must be displayed only if Qute LS is started (if user opens a Qute template or a java file).